### PR TITLE
Add issue testing UI flow

### PR DESCRIPTION
**Pull request by sebastinez**
This PR adds a test that checks the UI from
- Authentication with a `radicle-httpd` instance
- Creation of a new issue on a fresh created project
- Edit the title of the created issue
- Write comments and replies to the comment
- Cycle through the different states of an issue

Pending:
- Will add the `authenticatedPeer` fixture from #841

### CODE REVIEW

File: tests/e2e/project/issues.spec.ts [hunk 1]
```diff
@@ -55,3 +55,53 @@ test("test issue editing failing", async ({ page, authenticatedPeer }) => {
...
+  await page.getByRole("button", { name: "edit" }).first().click();
+  await page.getByPlaceholder("Title").fill("This is a new title");
+  await page.getByRole("button", { name: "save" }).click();
+
+  await page.getByPlaceholder("Leave your comment").fill("This is a comment");
+  await page.getByRole("button", { name: "Comment" }).click();
+
+  await page.getByRole("button", { name: "reply" }).click();
+  await page.getByPlaceholder("Leave your reply").fill("This is a reply");
+  await page.getByRole("button", { name: "Reply", exact: true }).click();
```
Comment (rudolfs) [inline tests/e2e/project/issues.spec.ts:95]: We should add some `expect`s for the title, description, assignee, tag, new title, comment and comment reply.

File: tests/e2e/project/issues.spec.ts [hunk 2]
```diff
@@ -55,3 +55,65 @@ test("test issue editing failing", async ({ page, authenticatedPeer }) => {
...
+  await page.getByRole("link", { name: "0 issues" }).click();
+  await page.getByRole("link", { name: "New issue" }).click();
+  await page.getByPlaceholder("Title").fill("This is a title");
+  await page
+    .getByPlaceholder("Write a description")
+    .fill("This is a description");
+
+  await page.getByPlaceholder("Add assignee").fill(authenticatedPeer.nodeId);
+  await page.getByPlaceholder("Add assignee").press("Enter");
+  await expect(
```
Comment (rudolfs) [inline tests/e2e/project/issues.spec.ts:75]: Let's move this to after we save the issue.

File: tests/e2e/project/issues.spec.ts [hunk 3]
```diff
@@ -55,3 +55,65 @@ test("test issue editing failing", async ({ page, authenticatedPeer }) => {
...
+      `did:key:${authenticatedPeer.nodeId.substring(
+        0,
+        6,
+      )}…${authenticatedPeer.nodeId.slice(-6)}`,
+    ),
+  ).toBeVisible();
+
+  await page.getByPlaceholder("Add tag").fill("bug");
+  await page.getByPlaceholder("Add tag").press("Enter");
+  await expect(page.getByText("bug")).toBeVisible();
```
Comment (rudolfs) [inline tests/e2e/project/issues.spec.ts:86]: And this one too.

File: tests/e2e/project/issues.spec.ts [hunk 4]
```diff
@@ -55,3 +55,65 @@ test("test issue editing failing", async ({ page, authenticatedPeer }) => {
...
+  await page.getByRole("button", { name: "Reply", exact: true }).click();
+  await expect(page.getByText("This is a reply")).toBeVisible();
+
+  await page.getByRole("button", { name: "Close issue as solved" }).click();
+  await expect(page.getByText("closed as solved")).toBeVisible();
+
+  await page.getByRole("button", { name: "Reopen issue" }).click();
+  await expect(page.getByText("open", { exact: true })).toBeVisible();
+
+  await page.locator('button[name="stateToggle"]').click();
```
Comment (rudolfs) [inline tests/e2e/project/issues.spec.ts:115]: Can we use the `getByRole` syntax here?

Comment (sebastinez) [reply tests/e2e/project/issues.spec.ts]: Yeah wasn't able to, I did change the `<button name="stateToggle" ..>` to `<button aria-label="stateToggle" ..>` which allows us to use `getByRole`
Seems like the `name` property in `getByRole` for a `button` isn't a `name` attribute but a label or the content of the button

**Review by rudolfs — APPROVED** [review]

**Pull request closed without merging**